Project/src/MakeCall/CallCard.js

```diff
@@ -516,6 +544,16 @@ export default class CallCard extends React.Component {
         }
     }
 
+    async handleRaiseHand() {
+        try {
+            this.state.isHandRaised ?
+                this.raiseHandFeature.lowerHand():
+                this.raiseHandFeature.raiseHand();
+                this.setState({isHandRaised: utils.isParticipantHandRaised(this.identifier, this.raiseHandFeature.getRaisedHands())})
```

Comment:
is this line needed, can we remove it? I would expect the call to `raiseHand()` or `lowerHand()` to trigger the raiseHandChangedHandler which will handle updating state